management/util.go

```diff
@@ -5,7 +5,9 @@ import (
 	"net/http"
 )
 
-func getResponseBody(response *http.Response) ([]byte, error) {
-	defer response.Body.Close()
+func getResponseBody(response *http.Response) (out []byte, err error) {
+	defer func() {
```

Comment:
I'm not super ok with this one 1) `response.Body` can be nil 2) returning `Close()` error is a behavior change. also why do you add named returns?